pkg/virt-launcher/virtwrap/agent-poller/agent_parser.go

```diff
@@ -196,6 +197,7 @@ func parseFilesystem(agentReply string) ([]api.Filesystem, error) {
 			Type:       fs.Type,
 			TotalBytes: fs.TotalBytes,
 			UsedBytes:  fs.UsedBytes,
+			Disk:       fs.Disk,
```

Comment:
Is it safe to just assign it like that w/o iterating and running append` to the list you create with a `make`?

Reply:
Yeah I think it's safe to just assign slices